src/backend/distributed/README.md

```diff
@@ -1722,17 +1722,20 @@ Merge command the same principles as INSERT .. SELECT processing. However, due t
 
 # DDL
 
-DDL commands are primarily handled via the ProcessUtility hook, which gets the parse tree of the DDL command. For supported DDL commands, we always follow the same sequence of steps:
+DDL commands are primarily handled via the citus_ProcessUtility hook, which gets the parse tree of the DDL command. For supported DDL commands, we always follow the same sequence of steps:
 
 1. Qualify the table names in the parse tree (simplifies deparsing, avoids sensitivity to search_path changes)
 2. Pre-process logic
-3. Call original ProcessUtility to execute the command on the local shell table
+3. Call original previous ProcessUtility to execute the command on the local shell table
 4. Post-process logic
 5. Execute command on all other nodes
 6. Execute command on shards (in case of table DDL)
 
 Either the pre-process or post-process step generates a "Distributed DDL Job", which contains a task list to run in steps 4 & 5 (via adaptive executor).
 
+
+> :warning: If both pre-process and post-process returns a task list, only the post-process job is executed. The pre-process job is ignored.
```

Comment:
This feels like a bug (or missing assert), so maybe remove this and add Assert to the code :)

Reply:
Actually an assert already exists. @onurctirtir realized and we changed this part. Please check for the latest version

Reply:
I think it at makes sense to say in this document that only one of them should return a task list. But I agree there should be an assert too.

Reply:
After reading further I noticed this was already mentioned below in the description of the preprocess & postprocess fields. I don't think this info needs to be present twice in this document. So let's remove it here and leave it in the part below.

Reply:
Removed